master.tex

```diff
@@ -1,19 +1,20 @@
-\documentclass[
-version=last,toc=bib,toc=graduated,toc=index,toc=listof,fontsize=9pt,openany]{scrbook}
+\documentclass[version=last,toc=bib,toc=graduated,toc=index,toc=listof,fontsize=9pt,openany,twoside]{scrbook}
```

Comment:
I though `twoside` isn't necessary because I define the margins manually (I calculated them manually back in 2014), is it?

Reply:
sorry, this was a left-over from something I tried out earlier. I will revert it.